File: conformance/cli-fixtures/sbt-multi/alpha/src/test/scala/alpha/AlphaTest.scala
package alpha

import java.nio.file.Files
import java.nio.file.Path
import org.junit.Assert.assertEquals
import org.junit.Test

class AlphaTest {
  @Test
  def value(): Unit = {
    Files.writeString(Path.of("affected-alpha.marker"), "alpha")
    assertEquals(1, new Alpha().value)
  }
}

File: conformance/cli-fixtures/ant/test/alpha/AlphaTest.java
package alpha;

import java.nio.file.Files;
import java.nio.file.Path;

public final class AlphaTest {
    public static void main(String[] args) throws Exception {
        if (!"alpha".equals(new Alpha().name())) {
            throw new AssertionError("alpha");
        }
        Files.writeString(Path.of("affected-alpha.marker"), "alpha");
    }
}
